holoviews/ipython/display_hooks.py

```diff
@@ -166,9 +167,10 @@ def element_display(element, max_frames):
         IPython.display.display(IPython.display.HTML(info))
         return
 
-
     backend = Store.current_backend
-    if type(element) not in Store.registry[backend]:
+    eltype = type(element)
+    if (eltype not in Store.registry[backend] and
+        all(eltype.__name__ != d.pattern for d in Compositor.definitions)):
```

Comment:
How come we didn't need this before when the compositor was used?

Reply:
I can actually remove this again, it was needed when I didn't define plotting class stubs for these elements, which I then restored because I realized options wouldn't work correctly without them.

Reply:
Ok, removing this would be good in that case.